db/import-tmpl/instance_db.yml

```diff
@@ -0,0 +1,40 @@
+# This file is a part of rAthena.
+#   Copyright(C) 2018 rAthena Development Team
+#   https://rathena.org - https://github.com/rathena
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###########################################################################
+# Custom Instance Database
+###########################################################################
+#
+# Instance Settings
+#
+###########################################################################
+# ID - Unique Instance ID.
+###########################################################################
+# Name - Instance Name.
```

Comment:
Some instance script commands are using **Name** as parameter. The **Name** should be unique too.